routines/analyze.go

```diff
@@ -0,0 +1,103 @@
+package routines
+
+import (
+	"log"
+	"time"
+
+	"github.com/csivitu/bl0b/ctftime"
+
+	"github.com/csivitu/bl0b/db"
+)
+
+func computeStatus(start time.Time, end time.Time) string {
+	t := time.Now().Unix()
+	startUnix := start.Unix()
+	endUnix := end.Unix()
+
+	if startUnix <= t && t < endUnix {
```

Comment:
Its a better idea to instead use `time.before` / `time.after` instead of the unix timestamp. As mentiond in the other comment, we can stick to using Time object everywhere, and just in the get Events function it gets converted to a unix one internally.

Reply:
I'll work on this, assuming we'll be doing it in O(n) instead of O(1) here: https://github.com/csivitu/bl0b/pull/6#discussion_r525942478

Reply:
This is done, resolving